.circleci/config.yml

```diff
@@ -160,21 +173,30 @@ commands:
             - project
             - .m2
   run-integration-tests:
+    parameters:
+      run-code-coverage:
+        default: false
+        type: boolean
+      fail-rerun-count:
+        default: 2
```

Comment:
We shouldn't change the behavior of re-running failed tests here - we're working on fixing the flappers, re-running them automatically hides real problems.

Reply:
Done, changed defaults to 0